source/common/tracing/http_tracer_impl.cc

```diff
@@ -501,16 +480,15 @@ void ZipkinReporter::flushSpans() {
 }
 
 void ZipkinReporter::onFailure(Http::AsyncClient::FailureReason) {
-  std::cerr << "Error posting spans to Zipkin" << std::endl;
+  // TODO: stats
```

Comment:
in case it helps, here's an example of another tracer's stats:

https://github.com/openzipkin/zipkin-finagle#metrics

most use cases are comparing against server stats:

https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md#collector